apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1056,8 +1089,13 @@ check_accept_msg(#{ chain_hash           := ChainHash
             {error, chain_hash_mismatch}
     end.
 
-dep_tx_for_signing(#{from := From} = Opts, #data{on_chain_id = ChanId, state=State}) ->
-    StateHash = aesc_offchain_state:hash(State),
+dep_tx_for_signing(#{from := From, amount := Amount} = Opts,
+                   #data{on_chain_id = ChanId, state=State}) ->
+    Updates = [aesc_offchain_state:op_deposit(From, Amount)],
+    UpdatedStateTx = aesc_offchain_state:make_update_tx(Updates, State, Opts),
+    {channel_offchain_tx, UpdatedOffchainTx} = aetx:specialize_type(UpdatedStateTx),
+    StateHash = aesc_offchain_tx:state_hash(UpdatedOffchainTx),
```

Comment:
I see [hash after the deposit](https://github.com/aeternity/protocol/blame/52de7f9a1e86428a9d6b729d72764f088372c440/epoch/api/channels_api_usage.md#L562).